### PR TITLE
fix(AppearanceView): UI issue fixes

### PR DESCRIPTION
**Pull request by caybro**
- fix system appearance preview being cut off
- bring the panel close to UI design (I haven't changed the common stuff like headline fonts, overall margins, etc)

Closes: #6623

### What does the PR do

Brings the Appearance Settings panel closer to the intended design

### Affected areas

AppearanceView

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

Light:
Snímek obrazovky z 2022-07-26 13-39-02

Dark:
Snímek obrazovky z 2022-07-26 13-39-07

Side by side with design:
Snímek obrazovky z 2022-07-26 13-42-53

### CODE REVIEW

**Review by stefandunca — APPROVED** [review]

**Review by osmaczko — APPROVED** [review]

File: ui/app/AppLayouts/Profile/views/AppearanceView.qml [hunk 1]
```diff
@@ -23,12 +23,6 @@ SettingsContentBase {
 
     property var systemPalette
 
-    enum Theme {
-        Light,
-        Dark,
-        System
-    }
```
Comment (PascalPrecht) [inline ui/app/AppLayouts/Profile/views/AppearanceView.qml:30]: Just curious, why was this removed? Doesn't look like there are any other changes there `Theme` needed to be removed. So I guess it was... just not used?

**Review by PascalPrecht — APPROVED** [review]

File: ui/app/AppLayouts/Profile/views/AppearanceView.qml (hunk 1 above)
Comment (caybro) [reply ui/app/AppLayouts/Profile/views/AppearanceView.qml:30]: Yeah, not used at all; the current theme is checked like this:
`control.checked: localAppSettings.theme === Universal.System`
